content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4/index.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: 'Upgrade to Prisma 4'
+metaTitle: 'Upgrade to Prisma 4'
+metaDescription: 'Guides on how to upgrade to Prisma 4'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+Prisma 4 introduces a number of **breaking changes** if you are upgrading from an earlier version (`3.x.x`). This guide explains how this upgrade might affect your application and gives instructions on how to handle any changes.
```

Comment:
This also applies to 2.x
```suggestion
Prisma 4 introduces a number of **breaking changes** if you are upgrading from an earlier version (`3.x.x` or `2.x.x`). This guide explains how this upgrade might affect your application and gives instructions on how to handle any changes.
```

Not sure the version number is needed at all though, could also be removed:
```suggestion
Prisma 4 introduces a number of **breaking changes** if you are upgrading from an earlier version. This guide explains how this upgrade might affect your application and gives instructions on how to handle any changes.
```

Reply:
Can you upgrade directly to 4 from 2 then, or do you have to go through the instructions in the 3 upgrade guide first?

Reply:
You should indeed also consider the upgrade guide for Prisma 3, but can do so with a P4 installed.

Reply:
Thanks. Changed to earlier version.